ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightApiResult.kt

```diff
@@ -0,0 +1,63 @@
+import org.json.JSONObject
+
+/**
+ * Represents the result from the Shopper Insight API.
+ *
+ * @property eligibleMethods Contains the payment methods available to the shopper.
+ */
+data class ShopperInsightApiResult(
+    val eligibleMethods: PaymentMethods
+) {
+    companion object {
+        fun fromJson(jsonString: String): ShopperInsightApiResult {
+            val jsonObject = JSONObject(jsonString)
+            val eligibleMethodsJson = jsonObject.getJSONObject("eligible_methods")
+            return ShopperInsightApiResult(PaymentMethods.fromJson(eligibleMethodsJson))
+        }
+    }
+}
+
+/**
+ * Contains details about the available payment methods.
+ *
+ * @property paypal Details about the PayPal payment method.
+ * @property venmo Details about the Venmo payment method.
+ */
+data class PaymentMethods(
```

Comment:
Nit - let's move each class to it's own file, that's typically the pattern we follow in the SDK currently

Reply:
Also let's prefix all of these with `ShopperInsights`

Reply:
fixed